sw/host/opentitanlib/src/app/mod.rs

```diff
@@ -224,8 +224,10 @@ pub struct TransportWrapper {
     strapping_conf_map: HashMap<String, HashMap<String, PinConfiguration>>,
     // Below fields are lazily populated, as instances are requested.
     pin_instance_map: RefCell<HashMap<String, Rc<gpio::GpioPinWrapper>>>,
-    spi_instance_map: RefCell<HashMap<String, Rc<spi::PhysicalSpiWrapper>>>,
-    i2c_instance_map: RefCell<HashMap<String, Rc<i2c::PhysicalI2cWrapper>>>,
+    spi_physical_map: RefCell<HashMap<String, Rc<spi::PhysicalSpiWrapper>>>,
```

Comment:
These fields needs  comments as to why we need to remember the SPI or I2C device in the session (e.g. for optimization and correctness)

Reply:
Done